gtk/rgfetchprogress.cc

```diff
@@ -133,18 +133,6 @@ RGFetchProgress::RGFetchProgress(RGWindow *win)
    // we need to make the window show before we obtain the _gc
    int id = _config->FindI("Volatile::PlugProgressInto", -1);
    //cout << "Plug ID: " << id << endl;
-   if (id > 0) {
-#if !GTK_CHECK_VERSION(3,0,0)
-      gtk_widget_hide(GTK_WIDGET(gtk_builder_get_object(_builder, "window_fetch")));
-      GtkWidget *vbox = GTK_WIDGET(gtk_builder_get_object(_builder, "vbox_fetch"));
-      _sock =  gtk_plug_new(id);
-      gtk_widget_reparent(vbox, _sock);
-      gtk_widget_show_all(_sock);
-      _win = _sock;
-#else
-      g_error("gtk_plugin_new not supported with gtk3");
```

Comment:
Similar as above, I think we should either keep the "g_error" here or we drop the "int id = " too because it is only needed for this. Ideally we would also document that "Volatile::PlugProgressInto" is no longer working but given that it was only working with gtk2 anyway and that gets dropped I doubt it's an issue :)